astropy/modeling/tests/test_parameters.py

```diff
@@ -534,21 +534,21 @@ def test_two_model_2d_array_parameters(self):
     def test_two_model_nonzero_model_set_axis(self):
         # An example where the model set axis is the *last* axis of the
         # parameter arrays
-        coeff = np.array([[[10, 20], [30, 40]], [[50, 60], [70, 80]]])
+        coeff = np.array([[[10, 20, 30], [30, 40, 50]], [[50, 60, 70], [70, 80, 90]]])
         coeff = np.rollaxis(coeff, 0, 3)
-        e = np.array([[1, 2], [3, 4]])
+        e = np.array([[1, 2, 3], [3, 4, 5]])
         e = np.rollaxis(e, 0, 2)
-        t = TParModel(coeff, e, model_set_axis=-1)
+        t = TParModel([coeff], [e], n_models=2, model_set_axis=-1)
```

Comment:
What is the purpose of bracketing the parameter values? Isn't this adding an extra dimension (increasing  it from 3 to 4)? The following tests still seem to expect 3 (or am I missing some magic?)

Reply:
Yes, this increases the dimension. I removed it and will push shortly.